scripts/setup/setup.cfg

```diff
@@ -1,6 +1,7 @@
-VHIVE_BRANCH='hy'
+VHIVE_BRANCH='main'
 LOADER_BRANCH='main'
 CLUSTER_MODE='container' # choose from {container, firecracker, firecracker_snapshots}
 PODS_PER_NODE=240
 GITHUB_TOKEN=$HOME/.git_token_loader
-DEPLOY_PROMETHEUS=false
\ No newline at end of file
+DEPLOY_PROMETHEUS=false
+YQ_VERSION=4.27.3
```

Comment:
I think it's better move to separate file in the same folder, name it 'versions.cfg' and `source` it together with the this setup.cfg. No need to expose this version in setup, we won't change it too often.